static/src/javascripts/projects/common/modules/commercial/adfree/renew-adfree-status.js

```diff
@@ -0,0 +1,40 @@
+define([
+    'common/utils/ajax-promise',
+    'common/utils/storage'
+], function (
+    ajaxPromise,
+    storage
+) {
+    return {
+        /* jscs:disable disallowDanglingUnderscores */
+        renew : renew,
+        _handleResponse : handleResponse // expose for testing
+        /* jscs:enable */
+    };
+
+    function renew() {
+        ajaxPromise({
+            // Endpoint returns cookie, 'gu_adfree_user'
```

Comment:
`ajax.js` has a header warning people not to use it any more and to prefer `ajaxPromise`; I figured I'd just follow suit.